pandas/tests/test_generic.py

```diff
@@ -1471,6 +1471,22 @@ def test_describe_typefiltering_groupby(self):
         self.assertTrue(G.describe(include=['number', 'object']).shape == (22, 3))
         self.assertTrue(G.describe(include='all').shape == (26, 4))
 
+    # GH 11517
+    def test_describe_multi_index_df_column_names(self):
+        df = pd.DataFrame({'A': ['foo', 'bar', 'foo', 'bar', 'foo', 'bar', 'foo', 'foo'],
+                           'B': ['one', 'one', 'two', 'three', 'two', 'two', 'one', 'three'],
+                           'C': np.random.randn(8),
+                           'D': np.random.randn(8)})
+        # test for hierarchical index
```

Comment:
great ,use a blank line between different cases (e.g. before the comment).

move the isssue number inside the test, and provide a description of what this tests is doing